raiden/transfer/state_change.py

```diff
@@ -786,3 +789,54 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self.__eq__(other)
+
+
+@dataclass
+class ContractReceiveRecoverableFailedChannelBatchUnlock(
+        ContractReceiveRecoverableFailureStateChange,
+):
+    transaction_from: typing.Address
+    token_network_identifier: typing.TokenNetworkIdentifier
+    participant: typing.Address
+    partner: typing.Address
+    locksroot: typing.Locksroot
+    unlocked_amount: typing.TokenAmount
+    returned_tokens: typing.TokenAmount
+
+
+@dataclass
+class ContractReceiveRecoverableFailedChannelClosed(
+        ContractReceiveRecoverableFailureStateChange,
+):
+    transaction_from: typing.Address
+    token_network_identifier: typing.TokenNetworkID
```

Comment:
here you use `typing.TokenNetworkID`, in the class above `typing.TokenNetworkIdentifier`. We should probably remove one from the codebase